src/main/webapp/app/pages/curation/GeneListPage.tsx

```diff
@@ -80,51 +95,61 @@ const GeneListPage = (props: StoreProps) => {
     },
   ];
 
+  const sidebarTabs = [
+    {
+      title: 'Tools',
+      content: <GeneListPageToolsTab metaData={isGermline ? props.germlineMetaData : props.metaData} />,
+    },
+  ];
+
+  if (!isGermline) {
+    sidebarTabs.push({
+      title: 'Data Validation',
+      content: <CurationDataValidationTab />,
+    });
+    sidebarTabs.push({
+      title: 'History',
+      content: <ReviewHistoryTab />,
+    });
```

Comment:
We are not showing history?

Reply:
The history tab is only used to generate the README of changes for the data release.

Reply:
Yeah, but I would still prefer to see the history if it's possible.